src/app/components/header/login/login.component.css

```diff
@@ -0,0 +1,36 @@
+.panel-heading {
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+#forget {
+  min-width: 100px;
+  margin-left: auto;
+  cursor: pointer;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+.form-inline label {
+  padding-left: 10px;
+  margin: 0;
+  cursor: S;
```

Comment:
is this valid code? or was this written and forgot to be removed?

Reply:
Yes, forgot to remove. This was for some stuff i was trying to implement but later thought to keep it simple,